boredGame/GameScene.swift

```diff
@@ -53,14 +53,16 @@ class GameScene: SKScene {
             if let name = touchedNode.name {
                 if name == "leftButton" {
```

Comment:
You could override the volume up/down button instead of using the on screen buttons. 
https://stackoverflow.com/questions/48251788/override-volume-up-in-swift-ios-10-to-take-a-picture

Also could just tap on the whole screen to jump